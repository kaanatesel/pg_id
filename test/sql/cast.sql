SELECT CAST(mgid '507f191e810c19729de860ea' AS TEXT);
SELECT CAST('507f191e810c19729de860ea' AS mgid);

SELECT CAST(mgid '507f191e810c19729de860ea' AS TEXT) = TEXT '507f191e810c19729de860ea';