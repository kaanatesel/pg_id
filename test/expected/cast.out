SELECT CAST(mgid '507f191e810c19729de860ea' AS TEXT);
           text           
--------------------------
 507f191e810c19729de860ea
(1 row)

SELECT CAST('507f191e810c19729de860ea' AS mgid);
           mgid           
--------------------------
 507f191e810c19729de860ea
(1 row)

SELECT CAST(mgid '507f191e810c19729de860ea' AS TEXT) = TEXT '507f191e810c19729de860ea';
 ?column? 
----------
 t
(1 row)

